app/views/admin/attachments/_attachments.html.erb

```diff
@@ -2,37 +2,41 @@
 <p class="govuk-body"><%= link_to 'Reorder attachments', [:reorder, :admin, typecast_for_attachable_routing(attachable), Attachment], class: "govuk-link govuk-link--no-visited-state" %></p>
 <% end %>
 
-<%= render "govuk_publishing_components/components/table", {
-  head: [
-    {
-      text: "Title"
-    },
-    {
-      text: "Type"
-    },
-    {
-      text: "Attachment"
-    },
-    {},
-    {}
-  ],
-  rows: attachable.attachments.includes(:attachment_data).map do |attachment|
-    [
-      {
-        text: sanitize("#{attachment.title} #{tag.span("Uploading", class: 'govuk-tag govuk-tag--green') unless attachment.attachment_data.blank? || attachment.attachment_data.uploaded_to_asset_manager?}")
-      },
-      {
-        text: attachment.is_a?(HtmlAttachment) ? attachment.readable_type : attachment.readable_type.capitalize
-      },
-      {
-        text: link_to_attachment(attachment, preview: true, full_url: true, class: "govuk-link govuk-link--no-visited-state")
-      },
-      {
-        text: link_to('Edit', [:edit, :admin, typecast_for_attachable_routing(attachable), attachment.becomes(Attachment)], class: "govuk-link govuk-link--no-visited-state")
-      },
-      {
-        text: link_to("Delete", [:confirm_destroy, :admin, typecast_for_attachable_routing(attachable), attachment.becomes(Attachment)], class: "govuk-link govuk-link--no-visited-state gem-link--destructive")
-      }
-    ]
-  end
-} %>
+<ul class="govuk-list">
+  <% attachable.attachments.includes(:attachment_data).map.with_index(1) do |attachment, index| %>
+    <li class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <p class="govuk-body">
+          <strong>Title:</strong> <%= attachment.title %> <% unless attachment.attachment_data.blank? || attachment.attachment_data.uploaded_to_asset_manager? %><span class="govuk-tag govuk-tag--green">Uploading</span><% end %>
+        </p>
+        <p class="govuk-body">
+          <strong>Type: </strong><%= attachment.is_a?(HtmlAttachment) ? attachment.readable_type : attachment.readable_type.capitalize %>
+        </p>
+        <p class="govuk-body">
+          <strong>Attachment: </strong><%= link_to_attachment(attachment, preview: true, full_url: true, class: "govuk-link govuk-link--no-visited-state") %>
+        </p>
+        <% if attachable.allows_inline_attachments? %>
+          <%= render "govuk_publishing_components/components/copy_to_clipboard", {
+            label: tag.strong("Inline attachment markdown code:"),
+            copyable_content: "[AttachmentLink: #{attachment_component_params(attachment)[:id]}]",
+            button_text: "Copy Markdown"
+          } %>
+          <div class="app-view-edition-resource__markdown--padded">
+            <%= render "govuk_publishing_components/components/copy_to_clipboard", {
+              label: tag.strong("Publication box markdown code:"),
```

Comment:
I'm not sure this is the right labelling to use. I think we should at least try to be consistent with the Govspeak help sidebar, which calls it an "attachment as a callout box" and a "link to an attachment".

Or, how about we standardise on this terminology?

> **Embed this attachment:**
> ```
> [Attachment: example.pdf]
> ```
>
> **Link to this attachment:**
> ```
> [AttachmentLink: example.pdf]
> ```